Upload fails when provided file is too large

Hi everyone,

when providing a file which has base64 encoded more than 20000000 character, the upload will fail:

```
Reading certificate from location: ########
Reading BOM from location: ########\sbom.json
Starting BOM upload to Dependency Track at url: ########
Project with name '########' and version '1.0.0.0' will be created if it does not exist.
Finished task execution with error - Error: Uploading the BOM to Dependency Track Failed: 400 - Bad Request
##[error]Uploading the BOM to Dependency Track Failed: 400 - Bad Request
##[error]Exit code 1 returned from process: file name '########\node.exe', arguments '"########\task.js"'.
```

Executing the PUT request manually returns following error:

```json
{
    "status": 400,
    "title": "The provided JSON payload could not be mapped",
    "detail": "The BOM is too large to be transmitted safely via Base64 encoded JSON value. Please use the \"POST /api/v1/bom\" endpoint with Content-Type \"multipart/form-data\" instead. Original cause: String value length (20054016) exceeds the maximum allowed (20000000, from `StreamReadConstraints.getMaxStringLength()`) (through reference chain: org.dependencytrack.resources.v1.vo.BomSubmitRequest[\"bom\"])"
}
```

I expect when switching to a POST request with the data as form data, this request could work.

See: https://github.com/DependencyTrack/dependency-track/issues/3182

--- a/UploadBOM/src/dtrackClient.js
+++ b/UploadBOM/src/dtrackClient.js
@@ -19,7 +19,7 @@ class DTrackClient {
       request('/api/v1/bom', {
         ...this.baseOptions,
         method: 'POST',
-        body: {
+        formData: {
           "project": projId,
           "bom": bom.toString()
         }

--- a/UploadBOM/src/dtrackClient.js
+++ b/UploadBOM/src/dtrackClient.js
@@ -18,10 +18,10 @@ class DTrackClient {
     return new Promise((resolve, reject) => {
       request('/api/v1/bom', {
         ...this.baseOptions,
-        method: 'PUT',
+        method: 'POST',
         body: {
           "project": projId,
-          "bom": bom.toString('base64')
+          "bom": bom.toString()
         }
       },
         (error, response) => {
@@ -38,12 +38,12 @@ class DTrackClient {
     return new Promise((resolve, reject) => {
       request('/api/v1/bom', {
         ...this.baseOptions,
-        method: 'PUT',
-        body: {
-          "autoCreate": true,
+        method: 'POST',
+        formData: {
+          "autoCreate": 'true',
           "projectName": name,
           "projectVersion": version,
-          "bom": bom.toString('base64')
+          "bom": bom.toString()
         }
       },
         (error, response) => {

